tmt/steps/provision/mock.py

```diff
@@ -487,15 +487,28 @@ def execute(
         interactive: bool = False,
         on_process_start: Optional[OnProcessStartCallback] = None,
         on_process_end: Optional[OnProcessEndCallback] = None,
+        sourced_files: Optional[list[Path]] = None,
         **kwargs: Any,
     ) -> tmt.utils.CommandOutput:
         """
         Execute command inside mock
         """
 
+        sourced_files = sourced_files or []
+
         if self.mock_shell.mock_shell is None:
             self.mock_shell.enter_shell()
 
+        for file in reversed(sourced_files):
```

Comment:
`reversed()` is used here, but not in `local` which seems to be using the same approach of composing the final command. Is that expected? Which of them is correct?

Reply:
`reversed()` is the correct one